docs/managing-data-quality/testing-and-qa-workflows/set-up-automated-testing-with-snowplow-micro/index.md

```diff
@@ -181,7 +186,7 @@ An example of a response with one event can be found below:
       "name_tracker":"sp1",
       "v_tracker":"js-3.1.3",
       "v_collector":"ssc-2.3.1-stdout$",
-      "v_etl":"snowplow-micro-1.3.1-common-2.0.2",
+      "v_etl":"snowplow-micro-${versions.snowplowMicro}-common-2.0.2",
```

Comment:
will the `common-2.0.2` part change over time?

Reply:
I'm thinking now that it probably doesn't make a huge amount of sense to use the variable here. Since it's an example of an event that's been copied and pasted, bumping the micro version in the payload seems a bit strange to me now.

Reply:
Good point! Overlooked that. I guess it should be reverted then.